src/routes/user/userparentalcontrol.tsx

```diff
@@ -223,7 +223,8 @@ const UserParentalControl: FunctionComponent = () => {
                 throw new Error('Unexpected null user.Policy');
             }
 
-            user.Policy.MaxParentalRating = parseInt((page.querySelector('#selectMaxParentalRating') as HTMLSelectElement).value || '0', 10) || null;
+            const parentalRating = parseInt((page.querySelector('#selectMaxParentalRating') as HTMLSelectElement).value || '0', 10);
+            user.Policy.MaxParentalRating = Number.isNaN(parentalRating) ? null : parentalRating;
```

Comment:
I would write:
```js
user.Policy.MaxParentalRating = parseInt((page.querySelector('#selectMaxParentalRating') as HTMLSelectElement).value || '0', 10) || null;
```

Reply:
That's how it was before. When checking it in the debugger it did not provide the intended behaviour since parseInt returns `0` which is evaluated as `false` leading to `MaxParentalRating` being set to `null` instead of `0`.

Reply:
Ah, right.